test/connectionStore.test.ts

```diff
@@ -576,5 +576,69 @@ suite('ConnectionStore tests', () => {
         });
     });
 
+    test('addSavedPassword not lookup password if password already set', (done) => {
+        // Given a profile with emptyPasswordInput set
+        let passwordAlreadySetProfile = Object.assign(new ConnectionProfile(), {
+            profileName: 'defaultNamedProfile',
+            server: 'namedServer',
+            database: 'bcd',
+            authenticationType: utils.authTypeToString(interfaces.AuthenticationTypes.SqlLogin),
+            user: 'cde',
+            password: 'asdf!@#$'
```

Comment:
hmm, I wonder if all the strings '...password..' in this file are going to open bugs